mangrove-core/how-to-guides/DirectHowTo.md

```diff
@@ -0,0 +1,307 @@
+# How to create a Direct contract
+
+This section will go through a few different ways of how a Direct contract could be implemented. If you don't know what a Direct contract is, we recommend reading both [MangroveOffer](../mangrover-offer.md) and [Direct](../Direct.md) before continuing.
+
+## Simple Direct implementation
+
+When creating a Direct contract, the first thing you need is to import the necessary contracts needed for the constructor. Next is creating a constructor that calls the constructor of Direct. We set the gas requirement to 30.000, since we are not doing much in the posthook<!-- FIXME: link to explanation of gas requirement and posthook -->, it does not need anymore than that.
+
+In the constructor we check if the deployer of the contract is the same as `msg.sender`, if not, it means that the sender is deploying the contract on behalf of another address and wants that address to be admin of the contract.
+
+```solidity
+pragma solidity ^0.8.10;
+
+pragma abicoder v2;
+
+import "mgv_src/strategies/offer_maker/abstract/Direct.sol";
+import "mgv_src/strategies/routers/AbstractRouter.sol";
+
+contract OfferMaker is Direct {
+
+  constructor(IMangrove mgv, AbstractRouter router_, address deployer) Direct(mgv, router_, 30_000) {
+    // stores total gas requirement of this strat (depends on router gas requirements)
+    // if contract is deployed with static address, then one must set admin to something else than msg.sender
+    if (deployer != msg.sender) {
+      setAdmin(deployer);
+    }
+  }
+}
+```
+
+Technically Direct does not require anything else. But since the newOffer function of Direct is an internal function, deploying the contract as-is, would not allow anyone to post an offer. Because of this we want to add one thing. We want to implement the IMakerLogic, which only says that the contract has to have a newOffer function with the correct parameters. You could choose to not use this interface, but it is a nice help, that enforces that the offer maker gives all the relevant information for posting a new offer. Since we only want the admin of the contract to be able to post offers, we add the modifier `onlyAdmin` to the function. This is a modifier Direct can use, because it is a `AccessControlled` contract.
```

Comment:
`_newOffer` not `newOffer`

Reply:
Also about implementing the `IMakerLogic` allows this strat to be compatible with the SDK who expects the IMakerLogic ABI

Reply:
Updated the text to reflect this